packages/survey/src/Question/ItemGroup/ItemGroup.tsx

```diff
@@ -0,0 +1,49 @@
+import { Box, useCheckboxGroup, useRadioGroup } from '@chakra-ui/react'
+
+import { Item } from '../Item'
+import { ItemProps, OptionObj } from '../Item/models/ItemProps'
+import { ItemGroupProps } from './'
+import { itemGroupStyle } from './style'
+
+/**
+ * TODO: Add Description
+ *
+ * My really useful description of the component.
+ */
```

Comment:
A meaningful description is missing here.

Reply:
done!